Mathlib/Geometry/Manifold/Instances/Real.lean

```diff
@@ -78,6 +77,12 @@ instance [NeZero n] : Inhabited (EuclideanHalfSpace n) :=
 instance : Inhabited (EuclideanQuadrant n) :=
   ⟨⟨0, fun _ => le_rfl⟩⟩
 
+instance {n : ℕ} [NeZero n] : Zero (EuclideanHalfSpace n) where
+  zero :=  ⟨fun _ ↦ 0, by norm_num⟩
```

Comment:
```suggestion
  zero.val := 0
  zero.prop := by norm_num
```
if you like (and same below)

Reply:
Why do *you* prefer it this way? (I don't have a strong opinion. I may argue that mathematically, not much is happening, so golfing it into one line feels fine.)